packages/cli/src/services/jwt.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Service } from 'typedi';
+import * as jwt from 'jsonwebtoken';
+
+@Service()
+export class JwtService {
```

Comment:
we should add some unit tests for this service, to prevent any future regressions in this bit of the code.